### PR TITLE
hotfix for slow deserialization

### PR DESCRIPTION
**Pull request by inabao**
fixed: #205

### CODE REVIEW

File: src/algorithm/hnswlib/hnswalg.h [hunk 1]
```diff
@@ -956,57 +956,9 @@ class HierarchicalNSW : public AlgorithmInterface<float> {
...
-        }
-
-        if (normalize_) {
-            size += max_elements_ * sizeof(float);
-        }
-
-        // output.close();
+        std::stringstream out_stream;
+        IOStreamWriter writer(out_stream);
+        size_t size = SerializeImpl(writer);
```
Comment (jiaweizone) [inline src/algorithm/hnswlib/hnswalg.h]: Here must do real serialize ?

**Comment by wxyucs** [comment]
@inabao Please create an issue and link to it.

File: src/algorithm/hnswlib/hnswalg.h (hunk 1 above)
Comment (inabao) [reply src/algorithm/hnswlib/hnswalg.h]: This side has just replicated the changes from the main branch. Future optimizations for this part will be made in the main branch.

**Review by jiaweizone — APPROVED** [review]
LGTM

**Pull request merged**